src/main/java/frc/auto/AutoBuilder.java

```diff
@@ -55,7 +55,8 @@ public void recordData() {
 
     public void convertData() {
         try {
-            writer.setFileName("test");
+            System.out.println("Made it to convertData()!");
```

Comment:
Is this needed for the code, or was that just for debugging, if debugging it should be removed from the code

Reply:
Again, debugging!